app/routes/profile.js

```diff
@@ -1,11 +1,19 @@
 import Route from 'ember-route';
 import service from 'ember-service/inject';
 import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+import RSVP from 'rsvp';
 
 export default Route.extend(AuthenticatedRouteMixin, {
   currentUser: service(),
-
   model() {
-    return this.get('currentUser.user');
+    return RSVP.hash({
+      user: this.get('currentUser.user'),
+    });
```

Comment:
`RSVP.hash` is typically used to return multiple pending operations from the model hook. Why are you using it to wrap only the user lookup?